README.md

```diff
@@ -28,37 +28,56 @@ A very basic Sculpin based blog supporting the following features:
 Install
 -------
 
+This application uses [Symfony's Webpack Encore](https://symfony.com/doc/current/frontend.html)
+to manage CSS, JavaScript and image assets. First install the JS dependencies:
+
+```bash
+$ yarn install
+```
+
+Then install the PHP dependencies (Sculpin):
+
 ```bash
 $ composer install
 ```
 
 Build
 -----
 
-    php vendor/bin/sculpin generate --watch --server
+First run Encore, to compile the assets in `source/assets/` into `source/build/`:
+
+```bash
+$ yarn encore dev
+```
+
+Then generate the site with Sculpin:
+
+```bash
+$ php vendor/bin/sculpin generate --watch --server
+```
 
 Your newly generated clone of sculpin-blog-skeleton is now
 accessible at `http://localhost:8000/`.
 
-Component Management
---------------------
-If you wish to install components via composer and have them automatically installed into your source directory you will need the following configuration options to your composer.json file.
+CSS, JavaScript and Images
```

Comment:
i'd keep the things above this in the readme so people can set up the project by following only the readme. but then move this thing into the basic-project.md page. there, the section will however need some intro that the skeleton comes with encore and `If you directly use sculpin/sculpin, follow the [setup instructions for encore](...)`

Reply:
Cool, I'm making a PR for sculpin/sculpin.io to do that now.

Reply:
See https://github.com/sculpin/sculpin.io/pull/120